xmeffects.js

```diff
@@ -287,3 +293,4 @@ var effects_t1 = [  // effect functions on tick 1+
   eff_unimplemented,  // y
   eff_unimplemented   // z
 ];
+})(window || {});
```

Comment:
Yep, and then there's `document` but anything that uses it has to be tested with some headless browser anyways. Which doesn't seem to be a very high priority in a project like this so we can just leave it as it is.

Adding the fallback to xm.js